server/routes/participant.js

```diff
@@ -181,13 +181,19 @@ participantsRouter.post(
   keycloak.getUserInfoMiddleware(),
   multer({
```

Comment:
possibly multer has input param `size: `

Reply:
I tried `limits` but I could not get it to reject the file :thinking:

Reply:
```    limits: { fileSize: 6000 },```